ton_sdk/Cargo.toml

```diff
@@ -3,11 +3,11 @@ name = "ton_sdk"
 version = "0.1.0"
 
 [dependencies]
-ton_abi = { git = "ssh://git@github.com/tonlabs/ton-labs-abi.git", branch = "private-repos" }
+ton_abi = { git = "ssh://git@github.com/tonlabs/ton-labs-abi.git", branch = "private-repos-2" } # Fix branch before merge!!!
```

Comment:
Fix branches. `ton_abi` and `ton_executor` are merged

Reply:
Done